### PR TITLE
added my favourite font :D

### PR DESCRIPTION
**Pull request by a7madgamal**
I hope the commit is ready

### CODE REVIEW

File: styles/fonts.less [hunk 1]
```diff
@@ -127,6 +127,10 @@
 .font ( 'Monoid', bold, normal, 'monoid/monoid-bold.ttf' );
 .font ( 'Monoid', normal, italic, 'monoid/monoid-italic.ttf' );
 .font ( 'Monoid', normal, normal, 'monoid/monoid.ttf' );
+.font ( 'Mononoki', bold, normal, 'mononoki/mononoki-Bold.ttf' );
+.font ( 'Mononoki', bold, italic, 'mononoki/mononoki-BoldItalic.ttf' );
```
Comment (braver) [inline styles/fonts.less]: `-Bold` etc. should be lowercase and `-bold-italic` needs to have a dash in between. I need the consistency to be able to reuse this across projects.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fonts",
   "main": "./lib/fonts",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (braver) [inline package.json]: You don't need to update this by hand, `apm` manages the version numbers upon release.

**Comment by braver** [comment]
Thanks! If you could have a look at my notes, I'll pull this after the corrections.

File: styles/fonts.less [hunk 3]
```diff
@@ -127,6 +127,10 @@
...
+.font ( 'Mononoki', normal, italic, 'mononoki/mononoki-Italic.ttf' );
+.font ( 'Mononoki', normal, normal, 'mononoki/mononoki-Regular.ttf' );
```
Comment (braver) [inline styles/fonts.less]: Please don't add `-Regular` to the regular font.

**Comment by a7madgamal** [comment]
done
I use
```
atom-workspace {
    font-family: Mononoki;
}
```
in my styles.less file to change the font for UI too, i think we should make this an option too or at least add it as a tip in the readme file

**Comment by braver** [comment]
It's usually not a good idea for packages to change theme styling. It's impossible to know how all themes have been styled and it's even harder to keep track of changes. However, I'm not opposed to adding a tip to the readme.

**Comment by a7madgamal** [comment]
I amended the last commit
all good?

**Comment by braver** [comment]
I think you need to update the filenames to lower case?

**Comment by a7madgamal** [comment]
I did!
other files than the 4 font files?

**Comment by braver** [comment]
screen shot 2016-05-25 at 16 37 31

**Comment by a7madgamal** [comment]
yeah I noticed now
something really weird is going on
git status says nothing to commit. finder also looks fine!!!
ss

**Comment by braver** [comment]
Weird.. are you on Windows?

**Comment by a7madgamal** [comment]
mac :D

**Comment by a7madgamal** [comment]
renamed inside atom. using Sourcetree for Git but never seen something like this before!

**Comment by braver** [comment]
Double weird. Well I guess it's fine then. I pull this and see if I need to make fixes before publishing. Thanks for the effort!

**Comment by a7madgamal** [comment]
uw 👍

**Comment by braver** [comment]
About the case change not being seen by git: https://www.garron.me/en/bits/git-ignorecase-rename-lowercase-uppercase.html

**Pull request merged**